Dockerfile

```diff
@@ -41,18 +46,15 @@ RUN mkdir /root/trainer/gin/datasets
 COPY ./trainer/gin/datasets/tfrecord.gin /root/trainer/gin/datasets/tfrecord.gin
 COPY ./trainer/gin/datasets/base.gin /root/trainer/gin/datasets/base.gin
 
-RUN mkdir /root/tmp
-RUN mkdir /root/tmp/ddsp
+# RUN mkdir /root/tmp
+# RUN mkdir /root/tmp/ddsp
 
+# These parameters can be also specified as part of the job submission arguments
 ENTRYPOINT ["python", "trainer/ddsp_run_multiple_vms.py", "--mode=train", \
      "--alsologtostderr", \
      "--save_dir=gs://werror-2020.appspot.com/mvp/dist", \
```

Comment:
Here and below. We should parametrize this. People shouldn't be seeing your personal bucket names in the variable contents.

Reply:
Done